README.rst

```diff
@@ -3,24 +3,24 @@ Haralyzer
 =========
 
 .. image:: https://badge.fury.io/py/haralyzer.svg
-    :target: http://badge.fury.io/py/haralyzer
+   :target: http://badge.fury.io/py/haralyzer
```

Comment:
Maybe go ahead and switch protocol to HTTPS while we are updating stuff?

Reply:
Good calll